Agent/AgentWar3/UnitRefList/Ability/AbilityInterfaced/CAbilityHero.h

```diff
@@ -16,31 +16,31 @@ class CAbilityHero : CAbilityInterfaced {
     int field_80;
     int field_84;
     int field_88;
-    int experience;
-    int skillPoints;
-    int strength; // assigned with the SetHeroStr native
-    int field_98;
-    int field_9C;
-    int field_A0;
-    int field_A4;
-    int agility; // assigned with the SetHeroAgi native
```

Comment:
Can you confirm this?

Reply:
No, the fields_## went missing somehow when i transfered from the old files and just enough fields was left to compensate.
This has been fixed in a new commit.